win32ss/w32ksvc.h

```diff
@@ -544,6 +544,9 @@
     SVC_(UserSetTimer, 4)
     SVC_(UserSetWindowFNID, 2)
     SVC_(UserSetWindowLong, 4)
+#ifdef _WIN64
+    SVC_(UserSetWindowLongPtr, 4)
```

Comment:
Is this the right syscall number, or is our syscall order totally broken on x64 anyway? :p